src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -99,13 +99,19 @@ public function onKernelResponse(ResponseEvent $event): void
             return;
         }
 
+        // do not capture redirects done in a non-conventional way (http-equiv="refresh" tag,...).
```

Comment:
are you actually returning a response using the HTTP status code of the redirection without the header providing the redirection location ?

`http-equiv="refresh"` is **not** a redirection (and so to me, it looks like it should be a 200 status code)

Reply:
> are you actually returning a response using the HTTP status code of the redirection without the header providing the redirection location ?`
> yes that's it!

```
// Metatag template redirection
if (!is_null($cachedLink->metatag)) {
    // As we need to have custom meta tags, we need an intermediary template
    // Redirection will be done either by browser or in JS
    $response = $this->render('app/redirects/metatag.html.twig', [
        'metatag' => $cachedLink->metatag,
        'redirectUrl' => $data['redirectUrl'],
    ]);

    $response->setStatusCode(Response::HTTP_FOUND);

    foreach ($data['headers'] as $key => $value) {
        $response->headers->set($key, $value);
    }

    return $response;
}
```
and the template has the `http-equiv="refresh"` tag.
for me it's because I develop an URL shortener, and this method allow to customize social media tags before the redirect.

Reply:
So maybe you have an idea on how to handle this on the Symfony side ? or maybe I'm wrong by doing this but it works well.
In all cases there is an exception in such case.

Reply:
I'm still wondering why you use a redirection status code in your response when you don't properly return a redirection (you return an HTML page that will trigger another navigation, which is *not* a HTTP redirection)